src/App.css

```diff
@@ -7,4 +7,20 @@
     min-height: 100vh;
     display: flex;
     flex-direction: column;
-  }
\ No newline at end of file
+  }
+  /* Profile */
+.edit-profile-back {
+  position: relative;
+}
+.edit-profile-icon {
+  position: absolute;
+  top: -30px;
+  right: -25px;
+  height: 100px;
+  width: 100px;
+  border-bottom-left-radius: 60%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 2;
+}
```

Comment:
Can't we do it that way?
https://getbootstrap.com/docs/5.1/utilities/position/

Reply:
everything that can be done in bootstrap has been moved to the className now